README.md

```diff
@@ -86,13 +92,17 @@ Parameters:
   a repository and can be resolved as a Gradle dependency.
 * `jdk` - the path to a JDK .tgz file.
 * `outputFile` - the path and file name of the output gzipped tar archive.
+* `signKeyChain (optional)` - the path and file name of the keychain which contains a code signing certificate.
+* `signPw (optional)` - the password which should be use to unlock the keychain.
+* `signAppID (optional)` - the application ID of the code signing certificate.
```

Comment:
I think this name is also misleading. Why is it called `AppId`? From what I can see in the code it's the "SIGNING IDENTITY" which is usually a developer identity obtained from apples developer program. I think we should simply call this `signIdentity` so that it's clear what to look for in the general code signing documentation. The documentation never mentions the term `AppId`.

Reply:
Changed property name to : `signIdentity`